wgkex/worker/netlink.py

```diff
@@ -49,12 +49,12 @@ def lladdr(self) -> str:
     @property
     def vx_interface(self) -> str:
         """Returns the name of the VxLAN interface associated with this lladdr."""
-        return f"vx-{self.domain}"
+        return f"vx{self.domain}"
```

Comment:
If we are changing the format here - we should be a bit careful, as we re-use this elsewhere. Could we consider putting these prefixes into the config file?